Config/IonAuth.php

```diff
@@ -1,4 +1,4 @@
-<?php namespace IonAuth\Config;
+<?php namespace Config;
```

Comment:
whats the reason for changing the namespacing? I like the idea of everything living under the IonAuth namespace, but this might be required for the config?

Reply:
> This is looking great. Left a couple comments. Looking through this, it wouldn't be difficult to make this a generic PHP project from this point.
If you're up for that, I think the main effort is injecting all the dependencies. Those dependencies can still be CI4 libraries. If you arent up for that, no worries, this will work for CI4 and we could cut a new version thats generic.

You're right. So I think, I have just to add **codeigniter4/framework**  in the require section in **composer.json**. Right ?

Reply:
I'm not sure since I've never tried this before with CI4...  It'd be best to include the exact libs we need and then inject them.  That'd also help with testing.

For example require CodeIgniter\Database\Database then inject it as a constructor param to the IonAuth model.

Reply:
Sorry, but I'm afraid it's beyond my skill :-(

Reply:
No worries!  We can always split off later, probably better to do it separate anyway.